service/implementation/auto_request_api/sport_data_managers/sport_consts.py

```diff
@@ -1,63 +1,97 @@
 from typing import Optional
+from enum import Enum
 
+class SportType(Enum):
+    handball = "handball"
+    hockey = "hockey"
+    afl = "afl"
+    rugby = "rugby"
+    baseball = "baseball"
+    basketball = "basketball"
+    volleyball = "volleyball"
+    football = "football"
+    nfl = "nfl"
+    mma = "mma"
+    formula_1 = "formula-1"
 
-def get_team_statistics_url(sport_name: str, team_id: Optional[int], league_id: Optional[int]):
-    print("Getting team statistics url")
-    match sport_name:
-        case "afl":
+def get_sport_type(sport_id: str) -> Optional[SportType]:
```

Comment:
Don't like the idea of having some specific methods in CONSTS file. Will have to discuss this